README.md

```diff
@@ -1,2 +1,28 @@
 # docker-osm-planet
 manage your copy of the OpenStreetMap planet file from within a docker container
+
+## build
+In this repository's root directory, run
+```shell
+docker build -t osm-planet .
+```
+
+## usage
+The container will download the latest OSM planet when run to its internal directory `/var/data/osm-planet`. Thus, mount a data-only container or a host directory to that location to receive the file:
+
+E.g., if you want the planet file inside `~/tmp/data-osm` on your host:
+```shell
+mkdir -p ~/tmp/data-osm
+docker run --rm -ti -v ~/tmp/data-osm:/var/data/osm-planet osm-planet
+```
+
+The `-ti` in there allows you to abort the download with **Ctrl-c**.
+The `--rm` removes the container when finished. (There's no good reason to keep it.)
+
+### advanced usage
+Pass the following environment variables to `docker run` with `--env <key>=<value>` to override the defaults:
+* **`osm_planet_mirror`** &mdash; OpenStreetMap planet mirror. See [OSM wiki](http://wiki.openstreetmap.org/wiki/Planet.osm#Planet.osm_mirrors) for available servers.
+* **`osm_planet_path_relative_to_mirror`** &mdash; the file to download from the mirror. Defaults to `pbf/planet-latest.osm.pbf`
+* **`osm_planet_dir`** &mdash; where to place the downloaded file inside the container. Defaults to `/var/data/osm-planet`. You'll want to mount a host direcotry or another container's volume to the location you specify here, so that you can retrieve the downloaded file when the osm-planet container has been removed after running.
+
+Do **not** override `osm_planet_script_dir`.
```

Comment:
Now that I think about it: Overriding `osm_planet_script_dir` for `run` wouldn't have any effect, as the variable isn't used during runtime. (And docker doesn't allow setting environment variables from the command line for `build`.) But this variable was rather useless anyway, so I've removed it in 4b2e7b6 (PR #2).